pkg/features/features.go

```diff
@@ -0,0 +1,25 @@
+package features
```

Comment:
Just a suggestion: Maybe we want to introduce a Repo Pattern here, in case we want to switch from Env Var to something different, e.g. an external Feature Flag Management or DB / Cache-Driven Flags?

Reply:
Tbh I do not know if it would make sense in our use-case, but we can definitive challenge it in future, but for now we need to disable the new Metrics asap, to get it promoted to STAGE